README.md

```diff
@@ -156,6 +156,89 @@ response:
 }
 ```
 
+### Create User account
+
+```js
+const rf = require('routefusion-sdk').Instance();
+
+let body = {
+  name_on_account: 'Bob TheBuilder',
+  account_number: '123',
+  routing_number: '1234',
+  currency: 'USD',
+  bank_country: 'US'
+};
+
+rf.createUserAccounts(body)
+  .then(resp => resp)
+  .catch(err => err)
+```
+response:
+```json
+{
+    "uuid": "d48cb8b3-8945-4748-9bed-kd3d9vc15m"
+}
+```
+
+### Get User accounts
+
+```js
+const rf = require('routefusion-sdk').Instance();
+
+// This only returns one account
+rf.getUserAccounts(userUuid)
+  .then(resp => resp)
+  .catch(err => err)
+```
+response:
+```json
```

Comment:
Is this supposed to be an array? You said it's an array of one, right?

Reply:
it is now :)